bigfix virtual current meter

pinkywafer opened the pull request:
closes #550

ledvinap commented:
@hydra : 38 includes to get one variable from master config. And tens of files to update if new master config structure is added ...

pinkywafer commented:
Updated battery_unittest.cc to deal with feature()  and calculateThrottleStatus().
I've added these as stubs... let me know if you want further tests on these instead

hydra commented:
Merged, thanks for the fix and thanks for your patience and updates to fix the dependency issues.
